Whiskey/Functions/Resolve-WhiskeyNuGetPackageVersion.ps1

```diff
@@ -22,10 +22,15 @@ function Resolve-WhiskeyNuGetPackageVersion
     if( -not $Version )
     {
         Set-Item -Path 'env:EnableNuGetPackageRestore' -Value 'true'
-        $Version = & $NugetPath list ('packageid:{0}' -f $NuGetPackageName) |
-                        Where-Object { $_ -match $NuGetPackageName } |
-                        Where-Object { $_ -match ' (\d+\.\d+\.\d+.*)' } |
-                        ForEach-Object { $Matches[1] }
+        $NuGetPackage = Invoke-Command -NoNewScope -ScriptBlock {
+            & $NugetPath list ('packageid:{0}' -f $NuGetPackageName)
+        }
+        $Version = $NuGetPackage |
+            Where-Object { $_ -match $NuGetPackageName } |
+            Select-Object -First 1 |

```

Comment:
Put Select-Object last. Just in case NuGet.exe changes its output on us, if `Select-Object` is last, we'll make sure only lines that have the package name and a version number are returned.

Reply:
Good idea. Done.